### ISSUE TITLE
Sector units options

### ISSUE THREAD

**Issue by em12509** [Turfjs/turf#1644]
Hi, I am getting the following error message when trying to implement the units options for a sector. (the same does work for buffer and lines).

```
Error: options is invalid
    at pe (turf.min.js?593c:formatted:1832)
    at Gn (turf.min.js?593c:formatted:3280)

```

The code is below where all the arguments are in the correct format as when I leave out the options argument, the sector works fine!

```
const options = {units: "miles"};
const sectorOuter = sector(centre.coordinates, radiusMax, b1, b2, options);
const sectorInner = sector(centre.coordinates, radiusMin, b1-1, b2+1, options);
```

Any help is very much appreciated, thanks!!!

**Comment by stebogit** [comment]
Looks like we have a 🐛. In fact we [never tested](https://github.com/Turfjs/turf/blob/master/packages/turf-sector/test.js) the `options` parameter.

```js
const options = {units: 'miles'};
const sectorOuter = sector(point([-75, 40]), 5, 25, 45, options);

// Error: options is invalid
//    at pe (turf.min.js?593c:formatted:1832)
//    at Gn (turf.min.js?593c:formatted:3280)

```

The faulty line is [this one](https://github.com/Turfjs/turf/blob/master/packages/turf-sector/index.js#L35). I can't explain what the problem is, as `isObject` does seem to return the correct value.

@em12509 what Turf version are you using?

**Comment by em12509** [comment]
I am on version 5.1.6 - thanks for getting back to me so fast!

**Comment by rowanwins** [comment]
Looking through the trace the problem is coming a few levels down. `Sector` calls `lineArc` which calls `destination`, so I think it's the call from `lineArc` to `destination` which is causing the grief.

But @stebogit is right that we ought to have a test for it.

From recollection I think I've resolved this already in the v7 branch which is available with the alpha tag on npm.

**Comment by em12509** [comment]
Great thanks!

**Comment by em12509** [comment]
When I "yarn add @turf/turf --upgrade", the latest version I get is 5.1.6 ... how can I force it to upgrade to v7?

**Comment by stebogit** [comment]
@em12509 I'm not sure, but have you tried `npm install @turf/turf@7.0.0-alpha.1` or `npm install @turf/turf@next`?
Ref https://docs.npmjs.com/cli/install

**Comment by em12509** [comment]
I keep getting an error No matching version found for @turf/turf@7.0.0-alpha.1 ... any ideas?

**Comment by rowanwins** [comment]
Gday @em12509 

try `npm install turf@7.0.0-alpha.1`

**Comment by em12509** [comment]
worked a treat - let's hope it solves the units issue! :)

**Comment by em12509** [comment]
Although it didn't actually update any of @turf/turf, it just added the following to the package.json

    "turf": {
      "version": "7.0.0-alpha.1",
      "resolved": "https://artifactory.palantir.build/artifactory/api/npm/all-npm/turf/-/turf-7.0.0-alpha.1.tgz",
      "integrity": "sha1-T0lIuWpOfiVFTBld8iG3GcFZX3c=",
      "requires": {
        "concaveman": "^1.1.1",
        "d3-voronoi": "^1.1.2",
        "density-clustering": "^1.3.0",
        "earcut": "^2.1.3",
        "geojson-equality": "^0.2.0",
        "geojson-rbush": "^3.1.1",
        "opencollective": "^1.0.3",
        "polygon-clipping": "^0.7.0",
        "rbush": "^2.0.2",
        "skmeans": "^0.9.7",
        "turf-jsts": "^1.2.3"
      }

but all the modules (buffer, sector etc) are still on v5.1.6

**Comment by EmersonBrSouza** [comment]
Hello, I'm getting the same problem :cry: . I'm trying to implement units options to @turf/circle. Do you found some solution for this problem?

**Comment by em12509** [comment]
I made the unit conversion myself in the end and kept the circle tool in km

**Comment by rowanwins** [comment]
Resolved since v6.2.0

**Comment by ChengDale** [comment]
I just meet the same problem
